Mathlib/Analysis/Calculus/FDeriv/Mul.lean

```diff
@@ -159,6 +159,69 @@ theorem fderiv_clm_apply (hc : DifferentiableAt 𝕜 c x) (hu : DifferentiableAt
 
 end ClmCompApply
 
+section ContinuousMultilinearApplyConst
+
+/-! ### Derivative of the application of continuous multilinear maps to a constant -/
+
+variable {ι : Type*} [Fintype ι]
+  {M : ι → Type*} [∀ i, NormedAddCommGroup (M i)] [∀ i, NormedSpace 𝕜 (M i)]
+  {H : Type*} [NormedAddCommGroup H] [NormedSpace 𝕜 H]
+  {c : E → ContinuousMultilinearMap 𝕜 M H}
+  {c' : E →L[𝕜] ContinuousMultilinearMap 𝕜 M H}
+
+theorem HasStrictFDerivAt.continuousMultilinear_apply_const (hc : HasStrictFDerivAt c c' x)
+    (u : ∀ i, M i) : HasStrictFDerivAt (fun y => (c y) u) (c'.flipMultilinear u) x :=
```

Comment:
*Personally*, I find `(u : (i : ι) → M i)` a bit clearer. Let's see what a more experienced reviewer says.

Reply:
It is written like this in other files (namely `Analysis.NormedSpace.Multilinear.Basic`)